labels.go

```diff
@@ -51,14 +51,14 @@ func (l Label) String() string {
 // ListLabels gets all labels for given project.
 //
 // GitLab API docs: https://docs.gitlab.com/ce/api/labels.html#list-labels
-func (s *LabelsService) ListLabels(pid interface{}, options ...OptionFunc) ([]*Label, *Response, error) {
+func (s *LabelsService) ListLabels(pid interface{}, opt *ListOptions, options ...OptionFunc) ([]*Label, *Response, error) {
```

Comment:
I try to prevent creating options structs that are randomly applicable to different functions. So where options are applicable, I always make sure there is a dedicated options struct. If the struct is the same it can just be a named type referencing the type you want. So in this case it could be written as:

```
// ListLabelOptions represents the available ListLabel() options.
//
// GitLab API docs: https://docs.gitlab.com/ce/api/labels.html#list-labels
type ListLabelOptions ListOptions
``` 

It's one additional line of code, but it makes things really clear IMO...

Reply:
Looks clean to me, only one problem tho. It doesn't seem to be the case everywhere in the library right now. Maybe I could do a PR to create a referenced dedicated type for each ListOption?

Reply:
Haha my comment is a bit late, just saw the PR you did to fix it all 👍

Reply:
Indeed... So if you can update this PR as well, then this one can be merged as well 👍